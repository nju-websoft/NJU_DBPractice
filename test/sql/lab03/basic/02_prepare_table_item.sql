open database db2025;
create table item (i_id int, i_im_id int, i_name char(8), i_price float, i_data char(10));
insert into item values (30, 0, '5X1A51WM', 0.60196, 'I9OG2IM94A');
insert into item values (90, 9, 'IDPN1L1T', 5.15894, '4JWHCAAZS4');
insert into item values (76, 6, '6WG66MA1', 7.62370, 'EJFXQCZ80L');
insert into item values (68, 33, '1DXFPUTN', 9.43778, 'SSARK8K167');
insert into item values (45, 15, 'NFXKOODL', 9.01165, '2ECSRX39IW');
insert into item values (43, 40, 'RWR0XRF5', 6.30719, '63FPBBAQND');
insert into item values (42, 18, 'YKGKQ3N5', 1.07642, 'CQ3F1TA3EM');
insert into item values (73, 39, 'CM5B88NR', 9.38108, 'CAR6HXC1WM');
insert into item values (29, 19, 'SWVTV38E', 4.68255, '6LHH8CH635');
insert into item values (25, 22, 'SXIGBROS', 6.88450, 'RYRW4QY2K6');
insert into item values (38, 24, 'J1YWIGK9', 3.71964, 'X4RQB1HODF');
insert into item values (48, 1, '3C3VKKCM', 5.83160, '5IWGP9EGTW');
insert into item values (56, 49, 'RUWDD1OH', 7.35659, 'SRM3J0GJAD');
insert into item values (64, 20, 'PPRCGEIH', 7.72220, '1PH3D3FQT5');
insert into item values (16, 17, 'IW8L55B6', 6.41740, 'TB2P222TWZ');
insert into item values (22, 3, '3ARJ0PFG', 1.94527, '44P0XR37UC');
insert into item values (62, 7, '1I3BM8UQ', 4.87226, '838MBK9AX9');
insert into item values (34, 45, '2FNRTFWX', 5.41399, 'FUEI7WWMO4');
insert into item values (23, 11, 'R8FQ4U4H', 1.03042, 'HPOMMX4MOY');
insert into item values (80, 18, 'HNMPR2C6', 2.62780, 'YGAWQXJR5E');
insert into item values (37, 20, '9F5F5YT8', 2.10391, 'OFHT187O90');
insert into item values (4, 47, '4NRGM36Z', 9.32881, 'W770BS0YDQ');
insert into item values (61, 3, '6P8EEPAH', 1.45761, 'TVT4R8SUI3');
insert into item values (75, 44, 'YT8D06U5', 6.12202, 'SIB8K1CXL9');
insert into item values (27, 44, 'O37BGIEN', 4.37474, 'PIQP5QNWDI');
insert into item values (15, 21, 'F860ROWK', 6.13980, '0FXS1H0MSG');
insert into item values (82, 10, 'IOFMH7OK', 2.65654, 'CQHJ9DX7ID');
insert into item values (91, 34, '7CN6VC6R', 0.56924, 'FN30JVWK0H');
insert into item values (18, 19, 'KDPI840W', 1.04260, 'FYNHKQ88SK');
insert into item values (20, 49, '8ZD45MBI', 8.59685, 'B7XO75OQ9D');
insert into item values (63, 13, 'GLMV6VRG', 9.51362, 'MERZEDMEVH');
insert into item values (9, 29, 'Q528R4ED', 1.20323, 'X0AAVVEERE');
insert into item values (12, 30, '2H1OOCUB', 9.81904, 'YPJSOGFFIB');
insert into item values (74, 19, 'C08BCO5P', 9.77577, 'N4SBEJRKRT');
insert into item values (31, 44, 'TWMBR33K', 1.54393, 'QW5WKSNR60');
insert into item values (84, 2, 'HBWNJTRN', 1.89624, 'B7EBH4UYVF');
insert into item values (6, 12, '40TINGH0', 4.93276, 'WLSHYJI2XH');
insert into item values (99, 41, 'TH4LSOCX', 7.27636, '7TRQPQH05E');
insert into item values (3, 33, 'KCFF7MHV', 1.90261, 'D6XGM7UUG9');
insert into item values (71, 12, '5K7I15W9', 7.66174, 'HAVK4P2TEH');
insert into item values (52, 32, 'LT70Q57Q', 3.72260, 'FUHDWY75CZ');
insert into item values (51, 11, 'PUEIOO6J', 1.99614, 'YUMJ3402XJ');
insert into item values (41, 28, 'WUGYBCHX', 4.57234, 'L28XYR80DC');
insert into item values (58, 36, 'PHDSVQZG', 2.12593, 'YDCQHYIVJC');
insert into item values (65, 40, 'JIZT7UEM', 1.72297, 'TB4Y07NAN1');
insert into item values (78, 4, 'Z9G4VRKD', 0.91686, 'VBJ200XLTR');
insert into item values (66, 2, 'HK6Y84WY', 7.55628, 'KAFSA86RZP');
insert into item values (21, 15, 'HPU0R1G2', 1.14042, 'F7N6BW2VUX');
insert into item values (86, 47, 'QV9GYNGP', 5.69364, 'ILEZ85YM7O');
insert into item values (57, 42, 'U5K70V0M', 0.44169, 'V3UA3E3XQD');
insert into item values (83, 26, '0WXHNWSL', 3.86874, 'UEWXKODBAX');
insert into item values (5, 24, 'K9U52F81', 0.06169, 'LUF0UM0723');
insert into item values (53, 9, '9AEH9AY7', 7.72782, 'XKFP2ARI6U');
insert into item values (47, 5, 'ALUB8FO0', 3.59033, '473JIRJUHI');
insert into item values (19, 2, '9CRMDMGG', 4.78935, 'LRHSW8LERZ');
insert into item values (36, 39, 'FPI64X9M', 5.39263, 'MO7E8WTPI6');
insert into item values (81, 21, 'XDKTHW28', 2.56057, 'LQ48H2NM8M');
insert into item values (72, 45, 'T6N26YC0', 5.37629, 'EU4DKK36JK');
insert into item values (49, 46, '2BIQ0JD0', 2.12296, 'NRM02MOIK5');
insert into item values (1, 16, 'GP2H3ARC', 2.02959, 'MJ04L44ZAA');
insert into item values (46, 16, '13YBQG19', 9.52102, '67V9ARV1QC');
insert into item values (24, 12, 'RILD3LD5', 5.82255, 'W7JMLWS0A2');
insert into item values (59, 22, 'JOJLXHSI', 1.17140, 'I75R388DI0');
insert into item values (69, 12, 'PW3IJQ4C', 1.91824, '6CV2ROEHNA');
insert into item values (35, 40, 'PW9JSCN3', 8.02289, '62MEOBMJPE');
insert into item values (96, 19, 'R34VC4MJ', 6.17394, '48V4MH200G');
insert into item values (8, 6, '7WOYXL75', 2.06200, '8F0N1GMRG1');
insert into item values (39, 47, 'IMLUQ22J', 1.85988, 'KGRKD419QM');
insert into item values (28, 35, 'O4BZO9SZ', 7.21039, 'OGH2OHV80M');
insert into item values (55, 48, 'D5U13GUK', 4.09440, 'RT2HXWVZL7');
insert into item values (67, 46, 'WY1C3NGG', 7.71385, 'A4Q9JGYD81');
insert into item values (26, 30, 'N5NWXHL7', 5.07828, 'XZ6PRGZ4CI');
insert into item values (93, 2, 'Z6KVZVO7', 3.95892, 'AWDXSCBNIS');
insert into item values (40, 41, 'DV02KYM0', 2.47840, '90F9WRJ9NG');
insert into item values (50, 13, 'QCKHI2ZU', 0.28637, '0Q28AD6H7L');
insert into item values (88, 42, 'PJ4CFFDL', 9.71816, 'CGG7CV3VP7');
insert into item values (54, 4, '9UA427TY', 0.99383, 'DTZYWT1VMI');
insert into item values (7, 46, 'YPPQU0C9', 1.96179, 'OVZSJCD89E');
insert into item values (97, 48, '90NZFMTK', 4.69616, 'V4H8MWGVKX');
insert into item values (11, 34, 'P8OSAWE9', 8.11813, 'XUNUGHH5S4');
insert into item values (32, 48, 'OILC2DGH', 3.08901, '800DJYVQNL');
insert into item values (94, 11, 'PXX6SD8N', 0.18113, 'B6CBHAQGWE');
insert into item values (13, 49, 'NWU9UM3U', 2.63509, 'JPOF880XAH');
insert into item values (44, 36, '6UGHBCDU', 0.12223, 'ZJ5YWQIW53');
insert into item values (92, 4, 'NEI1WXW5', 3.64392, '600A8GYOOZ');
insert into item values (79, 25, 'HA59L4FI', 3.23380, 'JHHH9QMNLZ');
insert into item values (95, 4, 'YK8FWITA', 0.88974, 'G99VA6Z8IT');
insert into item values (10, 2, 'LTJWOWDV', 6.59856, 'J7NY7K1AGT');
insert into item values (2, 3, 'RWMXL4XF', 4.08148, 'HHJO2VMYHQ');
insert into item values (89, 38, 'C6WXPVWV', 8.80908, 'QBFJFWEA6U');
insert into item values (17, 19, 'OV5AYM06', 6.72053, '6F0OBH19Y3');
insert into item values (33, 5, '0GYRGC1Y', 0.87208, 'OCRK5G0V00');
insert into item values (85, 48, 'I49QSH4J', 3.07684, 'EQNV40BI1C');
insert into item values (14, 7, 'LUELZUEL', 5.81724, 'J7WJ4ONCZ9');
insert into item values (0, 11, '7XCVGWGW', 3.41294, 'CRONME46AT');
insert into item values (70, 22, '0OBSD3TE', 5.28361, 'CRHN8WXN4D');
insert into item values (98, 17, 'RBNQVR42', 9.40795, '6TI6OU36QA');
insert into item values (60, 29, 'FATI6X07', 3.32567, 'UP1OAV7ZN5');
insert into item values (77, 16, 'VR795HJ8', 1.93842, 'JJTPSLX8YH');
insert into item values (87, 39, 'NHXKXPIH', 4.75606, '7AA0OAL0H3');
