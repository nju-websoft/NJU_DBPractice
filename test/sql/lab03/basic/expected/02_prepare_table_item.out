
+--------------+--------------+--------------+--------------+--------------+--------------+
| Database     | Table        | FieldNum     | RecordLength | StorageModel | IndexNum     | 
|              |              |              |              |              |              | 
+--------------+--------------+--------------+--------------+--------------+--------------+
| db2025       | item         | 5            | 30           | NARY_MODEL   | 0            | 
+--------------+--------------+--------------+--------------+--------------+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1

+--------------+
| inserted     | 
+--------------+
| 1            | 
+--------------+
Total tuple(s): 1
